applications/GeoMechanicsApplication/tests/cpp_tests/custom_geometries/test_interface_geometry.cpp

```diff
@@ -363,6 +374,22 @@ KRATOS_TEST_CASE_IN_SUITE(GetLocalCoordinatesOfAllNodesOfThreePlusThreeNodedLine
     KRATOS_EXPECT_MATRIX_NEAR(result, expected_result, 1e-6)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(AnyLineInterfaceGeometryHasTwoEdgesWithOppositeOrientations, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    const auto two_plus_two_noded_geometry     = CreateTwoPlusTwoNoded2DLineInterfaceGeometry();
+    const auto three_plus_three_noded_geometry = CreateThreePlusThreeNoded2DLineInterfaceGeometry();
+
+    const auto edges_of_2_plus_2_geometry = two_plus_two_noded_geometry.GenerateEdges();
+    const auto edges_of_3_plus_3_geometry = three_plus_three_noded_geometry.GenerateEdges();
+
+    KRATOS_EXPECT_EQ(edges_of_2_plus_2_geometry.size(), 2);
+    AssertNodeIdsOfGeometry(edges_of_2_plus_2_geometry(0), {1, 2});
+    AssertNodeIdsOfGeometry(edges_of_2_plus_2_geometry(1), {4, 3});
+    KRATOS_EXPECT_EQ(edges_of_3_plus_3_geometry.size(), 2);
+    AssertNodeIdsOfGeometry(edges_of_3_plus_3_geometry(0), {1, 2, 3});
+    AssertNodeIdsOfGeometry(edges_of_3_plus_3_geometry(1), {5, 4, 6});
+}
```

Comment:
Following the other tests (where 2+2 and 3+3 cases are tested separately), and considering there is no common setup for the two cases, I'd split this into two separate tests.

Reply:
The reason I've put them in a single test case is that the expected behavior is the same for all line interface geometries. But you rightly pointed out that we have the same split in all other test cases, too. So I have adopted your suggestion.